R/surv-roc_survival_auc.R

```diff
@@ -7,7 +7,7 @@
 #' @template return-dynamic-survival
 #' @details
 #'
-#' This method automatically groups by the `eval_time` argument.
+#' This method automatically groups by the `.eval_time` values.
```

Comment:
This is a dynamic metric. 

``` r
library(yardstick)

roc_auc_survival(
  lung_surv,
  truth = surv_obj,
  estimate = .pred
)
#> # A tibble: 1 × 3
#>   .metric          .estimator .estimate
#>   <chr>            <chr>          <dbl>
#> 1 roc_auc_survival standard       0.997
```

<sup>Created on 2023-04-01 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>